### PR TITLE
Dataflow v2

### PR DESCRIPTION
**Pull request by EagleoutIce**
The big update.

See https://github.com/Code-Inspect/flowr/milestone/6 for the current state.

### CODE REVIEW

File: src/dataflow/environments/environment.ts
Comment (Ellpeck) [inline src/dataflow/environments/environment.ts:1]: ur welcome <3

**Comment by EagleoutIce** [comment]
_This pull request is included in v2.0.0 (see [Release v2.0.0 (Dataflow v2)](https://github.com/Code-Inspect/flowr/releases/tag/v2.0.0))._

**Comment by EagleoutIce** [comment]
_This pull request is included in v2.0.0 (see [Release v2.0.0 (Dataflow v2)](https://github.com/Code-Inspect/flowr/releases/tag/v2.0.0))._

**Comment by EagleoutIce** [comment]
_This pull request is included in v2.0.0 (see [Release v2.0.0 (Dataflow v2)](https://github.com/Code-Inspect/flowr/releases/tag/v2.0.0))._

**Comment by EagleoutIce** [comment]
_This pull request is included in v2.0.0 (see [Release v2.0.0 (Dataflow v2)](https://github.com/Code-Inspect/flowr/releases/tag/v2.0.0))._

**Comment by EagleoutIce** [comment]
_This pull request is included in v2.0.0 (see [Release v2.0.0 (Dataflow v2)](https://github.com/Code-Inspect/flowr/releases/tag/v2.0.0))._

**Comment by EagleoutIce** [comment]
_This pull request is included in v2.0.0 (see [Release v2.0.0 (Dataflow v2)](https://github.com/Code-Inspect/flowr/releases/tag/v2.0.0))._

**Comment by EagleoutIce** [comment]
_This pull request is included in v2.0.0 (see [Release v2.0.0 (Dataflow v2)](https://github.com/Code-Inspect/flowr/releases/tag/v2.0.0))._

**Comment by EagleoutIce** [comment]
_This pull request is included in v2.0.0 (see [Release v2.0.0 (Dataflow v2)](https://github.com/Code-Inspect/flowr/releases/tag/v2.0.0))._

**Comment by EagleoutIce** [comment]
_This pull request is included in v2.0.0 (see [Release v2.0.0 (Dataflow v2)](https://github.com/Code-Inspect/flowr/releases/tag/v2.0.0))._

**Comment by EagleoutIce** [comment]
_This pull request is included in v2.0.0 (see [Release v2.0.0 (Dataflow v2)](https://github.com/Code-Inspect/flowr/releases/tag/v2.0.0))._

**Comment by EagleoutIce** [comment]
_This pull request is included in v2.0.0 (see [Release v2.0.0 (Dataflow v2)](https://github.com/Code-Inspect/flowr/releases/tag/v2.0.0))._

**Comment by EagleoutIce** [comment]
_This pull request is included in v2.0.0 (see [Release v2.0.0 (Dataflow v2)](https://github.com/Code-Inspect/flowr/releases/tag/v2.0.0))._

**Comment by EagleoutIce** [comment]
_This pull request is included in v2.0.0 (see [Release v2.0.0 (Dataflow v2)](https://github.com/Code-Inspect/flowr/releases/tag/v2.0.0))._

**Comment by EagleoutIce** [comment]
_This pull request is included in v2.0.0 (see [Release v2.0.0 (Dataflow v2)](https://github.com/Code-Inspect/flowr/releases/tag/v2.0.0))._

**Comment by EagleoutIce** [comment]
_This pull request is included in v2.0.0 (see [Release v2.0.0 (Dataflow v2)](https://github.com/Code-Inspect/flowr/releases/tag/v2.0.0))._

**Comment by EagleoutIce** [comment]
_This pull request is included in v2.0.0 (see [Release v2.0.0 (Dataflow v2)](https://github.com/Code-Inspect/flowr/releases/tag/v2.0.0))._

**Comment by EagleoutIce** [comment]
_This pull request is included in v2.0.0 (see [Release v2.0.0 (Dataflow v2)](https://github.com/Code-Inspect/flowr/releases/tag/v2.0.0))._

**Comment by EagleoutIce** [comment]
_This pull request is included in v2.0.0 (see [Release v2.0.0 (Dataflow v2)](https://github.com/Code-Inspect/flowr/releases/tag/v2.0.0))._

**Comment by EagleoutIce** [comment]
_This pull request is included in v2.0.0 (see [Release v2.0.0 (Dataflow v2)](https://github.com/Code-Inspect/flowr/releases/tag/v2.0.0))._

**Comment by EagleoutIce** [comment]
_This pull request is included in v2.0.0 (see [Release v2.0.0 (Dataflow v2)](https://github.com/Code-Inspect/flowr/releases/tag/v2.0.0))._

**Comment by EagleoutIce** [comment]
_This pull request is included in v2.0.0 (see [Release v2.0.0 (Dataflow v2)](https://github.com/Code-Inspect/flowr/releases/tag/v2.0.0))._

**Pull request merged**